lib/movie_reserver.rb

```diff
@@ -1,7 +1,104 @@
 require 'csv'
+require 'pry'
+require_relative 'date_range'
+require_relative 'movie'
+require_relative 'rental'
+
 
 module GreenBox
   class MovieReserver
+    attr_accessor :movies,:rentals
+    # :reserver, :date_range
+
+    def initialize
+      @movies = MovieReserver.load_movies
+      @rentals = []
+      @date_range = GreenBox::DateRange.new(Time.parse('2018-08-08'),Time.parse('2018-08-09'))
```

Comment:
This `@date_range` is unnecessary since there isn't one daterange needed by the system.  Instead each `Rental` will have it's own `DateRange`